notebooks/ProcessModelFitting.ipynb

```diff
@@ -72,15 +72,15 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "pymc      : 4.0.0b2\n",
```

Comment:
@michaelosthege The order was somehow changes here and in the other notebook, but the only change is the version number of the CatIB-TS package (0.3.0)